odahuFlow/operator/docs/docs.go

```diff
@@ -1,5 +1,6 @@
 // GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
 // This file was generated by swaggo/swag at
```

Comment:
Your sed doesn't work as expected. Please, revert changes in the docs.go, swagger.json and swagger.yaml files.